### DISCUSSION TITLE
Add support for sublayers / nested layers / layers within layers.

### DISCUSSION THREAD

**Discussion by KristofVDV** [jgraph/drawio#3999] [Ideas]
It would be nice to have support for nesting layers within layers, esp. when needing to hide/show a group of things that conceptually belong to different groups and subgroups. As such, those layers and sublayers could be used to represent those groups and subsgroups and used for showing or hiding them.

**Comment by yurisugano** [comment]
I came here to request this exact feature! We would benefit greatly from a nested diagram where we can "zoom in or out" of specific processes
